src/bin/vip-import-validate-files.js

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env node
+// @flow
+
+/**
+ * External dependencies
+ */
+import url from 'url';
+import path from 'path';
+import chalk from 'chalk';
+import fs from 'fs';
+import { promisify } from 'util';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import {
+	acceptedExtensions,
+	findNestedDirectories,
+	folderStructureValidation,
+	isFileSanitized,
+	doesImageHaveExistingSource,
+	logErrorsForIntermediateImages,
+	logErrorsForInvalidFileTypes,
+	logErrorsForInvalidFilenames,
+} from '../lib/vip-import-validate-files';
+
+// Promisify to use async/await
+const stat = promisify( fs.stat );
+const readDir = promisify( fs.readdir );
+
+command( { requiredArgs: 1, format: true } )
+	.example( 'vip import validate files <file>', 'Validate your media files' )
```

Comment:
```suggestion
	.command( 'files', 'Validate a media file/folder' )
	.example( 'vip import validate files <file>', 'Validate your media files' )
```
We want to add the import command to the help text here.

Reply:
Added in ce29959 and fixed the example to match the SQL validation example text!